container/bitvector/bitvector_test.go

```diff
@@ -19,6 +20,7 @@ func (s *BitVectorSuite) SetUpTest(c *C) {
 	data := []byte{0xF0, 0x0F}
 	length := 12
```

Comment:
just inline this

Reply:
I am not sure what you mean by this

Reply:
since length is only used immediately below, just do `s.vector = NewBitVector(data, 12)`

Reply:
Sure. In that case, shall I Inline the data arrays as well?

Reply:
s.vector = NewBitVector([]byte{0xF0, 0x0F}, 12)

	s.lVector = NewBitVector([]byte{0xF0, 0x0F}, 16)

Reply:
sure

Reply:
Cool :)